packages/editor-ui/src/hooks/utils/hooksAddAdminIcon.ts

```diff
@@ -0,0 +1,34 @@
+import { useUIStore } from '@/stores/ui.store';
+import type { IMenuItem } from 'n8n-design-system/types';
+import { useUsersStore } from '@/stores/users.store';
+
+let adminIconAdded = false;
+
+export const hooksAddAdminIcon = () => {
```

Comment:
this does not seem to up to date.. this would break cloud.. can you please bring this up to date? it needs to request a token from signin route and pass it to the new redirect path.. 
also would be great to move this in-app, since we already have similar logic for updates button...
related PR here https://github.com/n8n-io/n8n/pull/7316